src/app/window.cpp

```diff
@@ -60,8 +60,31 @@ void Dash::init()
     auto msg_ref = new QWidget();
     msg_ref->setObjectName("MsgRef");
     this->body.layout->addWidget(msg_ref);
-
+    this->rail.layout->addWidget(this->main_menu());
     this->body.layout->addWidget(this->control_bar());
+    this->set_page(this->arbiter.layout().curr_page);
+}
+
+void Dash::set_page(Page *page)
+{
+    bool fullscreen = this->arbiter.layout().fullscreen_mode;
+    int fullscreen_delay = this->arbiter.layout().fullscreen_delay;
+    if (fullscreen && fullscreen_delay != 0) {
+        this->arbiter.toggle_fullscreen_mode();
+        this->arbiter.toggle_fullscreen_mode();
```

Comment:
duplicate line here?

Reply:
yes, this is in purpose to trigger animation.

Reply:
i removed the duplicated line and it seemed to work just fine (tried with all animation levels)